deploy-manage/deploy/deployment-comparison.md

```diff
@@ -3,55 +3,55 @@
 
 This reference provides detailed comparisons of features and capabilities across Elastic's deployment options: self-managed deployments, Elastic Cloud Hosted, and Serverless. For a high-level overview of deployment types and guidance on choosing between them, see the [overview](../deploy.md).
```

Comment:
suggest linking to [serverless differences](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/577/deploy-manage/deploy/elastic-cloud/differences-from-other-elasticsearch-offerings#elasticsearch-differences-serverless-features-replaced) from this page